### PR TITLE
2022 EPA eGrid ingress

### PR DESCRIPTION
**Pull request by patstirling**
**Is this a new addition or an update to an existing factor?**

- [x] New
- [ ] Update

**What category or categories does it affect?**

Energy > Electricity

**What?**

US EPA have released their latest eGrid numbers - data provided here.

**Why?**

Top ensure latest US electricity grid emission factors are available

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -887,6 +940,33 @@ Energy,Electricity,electricity-energy_source_grid_mix_losses,Transmission & dist
...
+Energy,Electricity,electricity-energy_source_grid_mix_akgd,US eGRID subregion AKGD (ASCC Alaska Grid),kWh,not-supplied,0.500858,0.497882,0.001134,0.001892,not-supplied,not-supplied,,2,electricity_generation,EPA,2022,US,2022/25/02,Emission intensity of electicity in the US eGrid subregion as reported for 2022 based on 2020 statistics. Retrieved from the eGRID2020 Data File (xlsx) file published by the US EPA at the source URL.,https://www.epa.gov/egrid/download-data
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:943]: I think the region needs to reflect the egrid subregion and also be consistent with how it is specified in the OEFDB currently (e.g. row 736).

Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:943]: IDs are also not consistent with other EPA egrid EFs.

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -2079,8 +2159,8 @@ Transport,Air Freight,freight_flight-route_type_na-distance_na-weight_na-rf_na,A
...
 Transport,Air Freight,freight_flight-route_type_domestic-distance_na-weight_na-rf_included,Domestic air freight - with RF effect,tonne-km,4.97627427,4.977084,4.96965,0.00234,0.02475,not-supplied,not-supplied,,3,fuel_combustion,GHG Protocol,2021,GB,2022/01/15,Emission intensity of domestic air freight to/from UK including radiative forcing (RF) effect reflecting impact of contrails and high-altitude release of GHGs. The primary source is UK DEFRA Freighting goods 2019.,https://ghgprotocol.org/ghg-emissions-calculation-tool
+Transport,Air Freight,freight_flight-route_type_domestic-distance_na-weight_na-rf_included,Domestic air freight - with RF effect,tonne-km,,4.76719,,,,,,,,fuel_combustion,MfE,2020,NZ,2021/09/10,"Emission intensity of domestic freight flight in New Zealand including emissions from combustion and radiative forcing (RF) effect reflecting impact of contrails and high-altitude release of GHGs, but not upstream fuel emissions. Published by the Ministry for the Environment of New Zealand in 2020. Retrieved from 2020 Emission Factors Flat File.",https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:2163]: Would this cause any conflict? Same as 2190, 2198, 2784, and 3037.

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -295,14 +295,21 @@ Energy,Electricity,electricity-energy_source_grid_mix,Electricity generation –
...
 Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kWh,,0.4041,,,,,,,,electricity_generation,DEWA,2020,AE,2021/07/30,Emission intensity of electricity generated in the United Arab Emirates. Published by Dubai Electricity & Water Authority 2020.,https://www.dewa.gov.ae/en/consumer/Sustainability/sustainability-reports
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kWh,not-supplied,0.717,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,2/3,electricity_generation,GHG Protocol,2017,AE,2022/10/12,Emission intensity of grid mix as reported by GHG Protocol Scope 3 calculator. Originally retrieved from IEA 2011. Note that values are reported as CO2e by GHG Protocol and as CO2 by the original source.,https://quantis-suite.com/Scope-3-Evaluator/
```
[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:298]: Done!

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:2163]: Shouldn't do - no changes made to these lines and this passed ID checks last time

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: fixed - thanks!

**Review by maryarbab — APPROVED** [review]

**Pull request merged**